packages/cdk8s-cli/lib/import/crd.ts

```diff
@@ -106,6 +83,44 @@ export class ImportCustomResourceDefinition extends ImportBase {
   }
 }
 
+export class ImportCustomResourceDefinition extends ImportBase {
+  public static async match(source: string): Promise<undefined | CustomResourceApiObject[]> {
+    let manifest;
+    if (source.startsWith('https://')) {
+      manifest = await httpsGet(source);
+    } else if (path.extname(source) === '.yaml' || path.extname(source) === '.yml' || path.extname(source) === '.json') {
+      if (!(await fs.pathExists(source))) {
+        throw new Error(`can't find file ${source}`);
+      }
+
+      manifest = await fs.readFile(source, 'utf-8');
+    }
+
+  
+    if (!manifest) {
+      return undefined;
+    }
+
+    return yaml.parseAllDocuments(manifest).map((doc: yaml.ast.Document) => doc.toJSON());
```

Comment:
I feel like we should filter only the CustomResourceDefinition objects from the manifest, no? Also, add a test that verifies how this behaves when the manifest has more than just a CRD.

Reply:
I agree. Would we want some sort of message to users that we filtered out some objects from their manifest?